packages/odyssey-storybook/src/components/odyssey-labs/SideNav/SideNav.stories.tsx

```diff
@@ -454,3 +459,19 @@ export const CustomLogoImage: StoryObj<typeof SideNav> = {
     );
   },
 };
+
+export const DelayedSideNavItems: StoryObj<typeof SideNav> = {
```

Comment:
@benschell-okta Since we are adding a story for this, can we show the example with the correct loading state implemented as well?